EngineAPI/src/main/java/nullengine/event/world/item/ItemBreakEvent.java

```diff
@@ -0,0 +1,34 @@
+package nullengine.event.world.item;
+
+import nullengine.entity.Entity;
+import nullengine.event.Event;
+import nullengine.item.Item;
+import nullengine.world.World;
+
+public class ItemBreakEvent implements Event {
+	private World world;
+	private Entity player;
+	private Item item;
+	public ItemBreakEvent(Item item,World world, Entity player){
+		this.world=world;
+		this.item=item;
+		this.player=player;
+	}
+
+	@Override
+	public boolean isCancellable() {
+		return true;
```

Comment:
`isCancellable` return true don't mean this event is cancellable. Please implement interface `Cancellable`.